Model/Database/Updater.php

```diff
@@ -121,28 +143,37 @@ protected function loadDbContent($dbUrl)
      */
     protected function getDbUrl($dbCode)
     {
-        return str_replace('%db_name%', $this->database->getDbFileName($dbCode), $this->dbLocation);
+        $editionId = str_replace('.mmdb', '', $this->database->getDbFileName($dbCode));
+        $dbUrl = str_replace('%edition_id%', $editionId, $this->dbLocation);
+        $dbUrl = str_replace('%archive%', $this->dbArchiveExt, $dbUrl);
+        $dbUrl = str_replace('%license_key%', $this->webServiceConfig->getLicenseKey(), $dbUrl);
```

Comment:
maybe add a comment in the admin backend, that the webservice key is also used for downloading

Reply:
I've added a comment and updated the readme